### PR TITLE
eGrants automated test cases

### PR DESCRIPTION
**Pull request by hamidnesar**

### CODE REVIEW

File: src/main/resources/conf/env/stage.xml
Comment (iamdez99) [inline src/main/resources/conf/env/stage.xml:1]: Can you please add a comment above the application ID, for ex:

File: src/test/java/CustomBusiness/Egrants/Pages/EgrantsQuickLinkAndManagementMenuPage.java
Comment (iamdez99) [inline src/test/java/CustomBusiness/Egrants/Pages/EgrantsQuickLinkAndManagementMenuPage.java:1]: Can you please uppercase Commented element descriptions, for ex: Quick Links - QUICK LINKS

File: src/test/java/CustomBusiness/Egrants/Steps/QuickLinkVerificationAndManagementMenuSteps.java
Comment (iamdez99) [inline src/test/java/CustomBusiness/Egrants/Steps/QuickLinkVerificationAndManagementMenuSteps.java:1]: Great Job, Hamid.

File: src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java
Comment (iamdez99) [inline src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java:1]: Fantastic job, Hamid.

**Review by iamdez99 — CHANGES_REQUESTED** [review]
Please address comments.

File: src/test/java/CustomBusiness/Egrants/Pages/EgrantsQuickLinkAndManagementMenuPage.java
Comment (matakevin) [inline src/test/java/CustomBusiness/Egrants/Pages/EgrantsQuickLinkAndManagementMenuPage.java:1]: Remove lines 8, 23

Comment (hamidnesar) [reply src/test/java/CustomBusiness/Egrants/Pages/EgrantsQuickLinkAndManagementMenuPage.java:1]: Comments updated to uppercase.

File: src/main/resources/conf/env/stage.xml
Comment (hamidnesar) [reply src/main/resources/conf/env/stage.xml:1]: Added application ID

File: src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java
Comment (matakevin) [inline src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java:1]: remove lines 39, 133.  Aside from that, great coding!

**Review by matakevin — CHANGES_REQUESTED** [review]
Left some comment's regarding some empty lines but very good coding!

**Review by matakevin — APPROVED** [review]
LGTM!  Nice job!

**Review by puzikovam2 — APPROVED** [review]
approved by puzikovam2

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**